more_itertools/more.py

```diff
@@ -333,3 +333,12 @@ def intersperse(e, iterable):
             yield e
             yield item
     raise StopIteration
+
+
+# http://stackoverflow.com/a/13243870/2863746
+def empty_iterator():
+    """
+    An empty iterator.
+    """
+    return
```

Comment:
I like that solution better too (interestingly this was suggested on stackoverflow on the same day that I posted this patch!).